.github/workflows/go.yml

```diff
@@ -34,6 +34,9 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2.0.0
 
+    - name: Enter v2 directory
```

Comment:
we need to test v1 aswell

Reply:
v1 is on its own branch rather than master.